kibana/mapping.py

```diff
@@ -54,6 +55,19 @@ def pr_inf(self, msg):
     def pr_err(self, msg):
         print('[ERR] Mapping %s' % msg)
 
+    def get_map_ua(self):
```

Comment:
There are too many types for this single variable, it's a string, a custom ua obj, or a requests.session. I feel like the user agent concept is being blended with the sessions usage. Could we separate out or clarify those needs?